mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -1,47 +1,102 @@
 CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
-	COMMENT Represents a key for a value in a registry in a context where a
-	COMMENT root registry is available.
+	COMMENT A sugarcoated {@link Identifier} with a specific type of usage.
```

Comment:
```suggestion
	COMMENT A scoped {@link Identifier} that specifies the type of resource it refers to.
```

Reply:
I should call this a wrapped identifier I guess. Maybe rename the fields and such as well.

Reply:
I still think scoped/scope is probably better than wrapped since that's what I'd go to for describing this class. It scopes an identifier to a specific registry (it's technically not limited to registries though).